coding-concepts.js

```diff
@@ -62,7 +62,11 @@ class Learn {
   }
 }
 const learnStudent = new Learn("George")
-// console.log(learnStudent)
+ console.log(learnStudent)
 
-// a) Your answer:
-// b) Verify and explain:
\ No newline at end of file
+// a) Your answer: I am not quite sure because I have not made sense of classes at this time and the explanations I have received still do not help me understand this concept.
+// b) Verify and explain: I am having trouble grasping this particular concept and getting the information to stick. Perhapos I am overthinking it. I reference the notes and the first few things makes sense but when I look at the other examples I am completely lost.
+
+// I ran the code and it returned the following:
+// student:George, cohort: Bravo, year: 2023
+// My guess is when the new class "learnStudent" was created it used the parameter "George" to go through the information in the "Learn" class.
```

Comment:
So a class is like a blueprint for an object. And what we have here is we are saying use the class Learn to create a new object and for the student key, George will be the value. The others will stay the same, no change to Bravo and 2023 for cohort and year respectively.